package.json

```diff
@@ -61,6 +61,9 @@
     "type": "git",
     "url": "https://github.com/elastic/kibana.git"
   },
+  "resolutions": {
+    "**/@types/node": "8.10.20"
```

Comment:
The version specifier for this package is `^8.10.20` below, which is locked to `8.10.21` right now. That means that we currently have separate `8.10.20` for all dependencies, but a toplevel `8.10.21`:

```
$ yarn why @types/node
yarn why v1.10.1
[1/4] Why do we have the module "@types/node"...?
[2/4] Initialising dependency graph...
warning Resolution field "@types/node@8.10.20" is incompatible with requested version "@types/node@^9.4.7"
[3/4] Finding dependency...
[4/4] Calculating file sizes...
=> Found "@types/node@8.10.21"
info Has been hoisted to "@types/node"
info This module exists because it's specified in "devDependencies".
info Disk size without dependencies: "464KB"
info Disk size with unique dependencies: "464KB"
info Disk size with transitive dependencies: "464KB"
info Number of shared dependencies: 0
=> Found "@types/puppeteer#@types/node@8.10.20"
info This module exists because "@types#puppeteer" depends on it.
info Disk size without dependencies: "464KB"
info Disk size with unique dependencies: "464KB"
info Disk size with transitive dependencies: "464KB"
info Number of shared dependencies: 0
=> Found "@types/execa#@types/node@8.10.20"
info This module exists because "@types#execa" depends on it.
info Disk size without dependencies: "464KB"
info Disk size with unique dependencies: "464KB"
info Disk size with transitive dependencies: "464KB"
info Number of shared dependencies: 0
=> Found "@types/glob#@types/node@8.10.20"
info This module exists because "@types#glob" depends on it.
info Disk size without dependencies: "464KB"
info Disk size with unique dependencies: "464KB"
info Disk size with transitive dependencies: "464KB"
info Number of shared dependencies: 0
=> Found "@types/uuid#@types/node@8.10.20"
info This module exists because "@types#uuid" depends on it.
info Disk size without dependencies: "464KB"
info Disk size with unique dependencies: "464KB"
info Disk size with transitive dependencies: "464KB"
info Number of shared dependencies: 0
=> Found "@types/react-dom#@types/node@8.10.20"
info This module exists because "@types#react-dom" depends on it.
info Disk size without dependencies: "464KB"
info Disk size with unique dependencies: "464KB"
info Disk size with transitive dependencies: "464KB"
info Number of shared dependencies: 0
=> Found "@types/hapi-latest#@types/node@8.10.20"
info This module exists because "@types#hapi-latest" depends on it.
info Disk size without dependencies: "464KB"
info Disk size with unique dependencies: "464KB"
info Disk size with transitive dependencies: "464KB"
info Number of shared dependencies: 0
=> Found "@types/listr#@types/node@8.10.20"
info This module exists because "@types#listr" depends on it.
info Disk size without dependencies: "464KB"
info Disk size with unique dependencies: "464KB"
info Disk size with transitive dependencies: "464KB"
info Number of shared dependencies: 0
=> Found "@types/shot#@types/node@8.10.20"
info This module exists because "@types#hapi-latest#@types#shot" depends on it.
info Disk size without dependencies: "464KB"
info Disk size with unique dependencies: "464KB"
info Disk size with transitive dependencies: "464KB"
info Number of shared dependencies: 0
=> Found "@types/superagent#@types/node@8.10.20"
info This module exists because "@types#supertest#@types#superagent" depends on it.
info Disk size without dependencies: "464KB"
info Disk size with unique dependencies: "464KB"
info Disk size with transitive dependencies: "464KB"
info Number of shared dependencies: 0
=> Found "@types/iron#@types/node@8.10.20"
info This module exists because "@types#hapi-latest#@types#iron" depends on it.
info Disk size without dependencies: "464KB"
info Disk size with unique dependencies: "464KB"
info Disk size with transitive dependencies: "464KB"
info Number of shared dependencies: 0
=> Found "@slack/client#@types/node@8.10.20"
info This module exists because "x-pack#@slack#client" depends on it.
info Disk size without dependencies: "464KB"
info Disk size with unique dependencies: "464KB"
info Disk size with transitive dependencies: "464KB"
info Number of shared dependencies: 0
=> Found "@types/is-stream#@types/node@8.10.20"
info This module exists because "x-pack#@slack#client#@types#is-stream" depends on it.
info Disk size without dependencies: "464KB"
info Disk size with unique dependencies: "464KB"
info Disk size with transitive dependencies: "464KB"
info Number of shared dependencies: 0
=> Found "parse5#@types/node@8.10.20"
info This module exists because "enzyme#cheerio#parse5" depends on it.
info Disk size without dependencies: "464KB"
info Disk size with unique dependencies: "464KB"
info Disk size with transitive dependencies: "464KB"
info Number of shared dependencies: 0
=> Found "@types/form-data#@types/node@8.10.20"
info This module exists because "x-pack#@slack#client#@types#form-data" depends on it.
info Disk size without dependencies: "464KB"
info Disk size with unique dependencies: "464KB"
info Disk size with transitive dependencies: "464KB"
info Number of shared dependencies: 0
=> Found "@types/ws#@types/node@8.10.20"
info This module exists because "x-pack#@slack#client#@types#ws" depends on it.
info Disk size without dependencies: "464KB"
info Disk size with unique dependencies: "464KB"
info Disk size with transitive dependencies: "464KB"
info Number of shared dependencies: 0
=> Found "@types/got#@types/node@8.10.20"
info This module exists because "x-pack#@slack#client#@types#got" depends on it.
info Disk size without dependencies: "464KB"
info Disk size with unique dependencies: "464KB"
info Disk size with transitive dependencies: "464KB"
info Number of shared dependencies: 0
```

If we changed this to `8.10.21` too, the package would be hoisted:

```
$ yarn why @types/node
yarn why v1.10.1
[1/4] Why do we have the module "@types/node"...?
[2/4] Initialising dependency graph...
warning Resolution field "@types/node@8.10.21" is incompatible with requested version "@types/node@^9.4.7"
[3/4] Finding dependency...
[4/4] Calculating file sizes...
=> Found "@types/node@8.10.21"
info Has been hoisted to "@types/node"
info Reasons this module exists
   - Specified in "devDependencies"
   - Hoisted from "@types#puppeteer#@types#node"
   - Hoisted from "@types#execa#@types#node"
   - Hoisted from "@types#glob#@types#node"
   - Hoisted from "@types#uuid#@types#node"
   - Hoisted from "@types#react-dom#@types#node"
   - Hoisted from "@types#hapi-latest#@types#node"
   - Hoisted from "@types#listr#@types#node"
   - Hoisted from "@types#hapi-latest#@types#shot#@types#node"
   - Hoisted from "@types#supertest#@types#superagent#@types#node"
   - Hoisted from "@types#hapi-latest#@types#iron#@types#node"
   - Hoisted from "x-pack#@slack#client#@types#node"
   - Hoisted from "x-pack#@slack#client#@types#is-stream#@types#node"
   - Hoisted from "enzyme#cheerio#parse5#@types#node"
   - Hoisted from "x-pack#@slack#client#@types#form-data#@types#node"
   - Hoisted from "x-pack#@slack#client#@types#ws#@types#node"
   - Hoisted from "x-pack#@slack#client#@types#got#@types#node"
info Disk size without dependencies: "464KB"
info Disk size with unique dependencies: "464KB"
info Disk size with transitive dependencies: "464KB"
info Number of shared dependencies: 0
```

The same is true for `x-pack/package.json`.